### PR TITLE
Add Wilson Ng's Project Portfolio Page 

### PR DESCRIPTION
**Pull request by wilsonngja**

### CODE REVIEW

File: docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@startuml
+
+hide footbox
+
+Participant ":CommandFindClient" as CommandFindClient #F4F4B6
+Participant "clientList : ClientList" as ClientList #F4F4B6
+Participant "client : Client" as Client #F4F4B6
+Participant "ui: Ui" as Ui #F4F4B6
+
+->CommandFindClient:execute(Ui:ui, Storage: storage, PropertyList: propertyList, \nClientList: clientList, PairingList: pairingList
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml:10]: I think the format for arguments in method calls should be `methodName(<variable name>: <variable type>)`
```
execute(ui: Ui, storage: Storage, propertyList: PropertyList ...
```

File: docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        ClientList->Client:get(int: i)
+        activate Client #CDEFF9
+        return client
+        return currentClient
+
+        CommandFindClient->CommandFindClient:hasQueryText(String: queryText, Client: currentClient)
+        activate CommandFindClient #E2CDF9
+        CommandFindClient -> Client:getClientName()
+        activate Client #CDEFF9
+        return clientName
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/FindClientSequenceDiagram.puml:29]: I think the format for return should be `<variable name>: <variable type>`
```
clientName: String
```

File: docs/diagrams/PlantUML File/StorageAddClientSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+return
+
+Storage -> FileWriter:write(String: newText)
+activate FileWriter #CDEFF9
+return
+
+Storage -> FileWriter:close()
+activate FileWriter #CDEFF9
+return
+destroy FileWriter
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/StorageAddClientSequenceDiagram.puml:24]: Maybe you could use powerpoint to draw the destruction cross

File: docs/diagrams/PlantUML File/StorageUpdatePairSequenceDiagram.puml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+        Storage->Client:getClientEmail()
+        activate Client #CDEFF9
+        return clientEmail
+
+        Storage->Client:getClientBudgetPerMonth()
+        activate Client #CDEFF9
+        return clientBudgetPerMonth
+
+        Storage->ClientPropertyPair:get(Client: client)
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/StorageUpdatePairSequenceDiagram.puml:39]: There were multiple `clientPropertyPair.get(client)` calls in your code, which I don't think is reflected in your sequence diagram. Perhaps you can draw the multiple calls or refactor your code such that there is only one call.

File: docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+create FileWriter
+Storage -> FileWriter:FileWriter(PROPERTY_PATH)
+activate FileWriter #CDEFF9
+return
+
+Storage -> PropertyList:getPropertyList()
+activate PropertyList #CDEFF9
+return propertyList
+
+    loop for each property in propertyList
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml:23]: Do note that the loop condition should be a boolean statement

File: docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        Storage -> Property:getLandlordName()
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml:24]: In your code, there are multiple `propertyLists.get(i)` calls which are not reflected in your sequence diagram.
Maybe you can refactor your code to make a single call instead

Comment (ngdeqi) [reply docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml:24]: Also, the name `propertyLists` is a bit misleading as it implies that the variable contains multiple lists when it only stores a single property list

File: src/main/java/seedu/duke/Storage.java [hunk 7]
```diff
@@ -684,17 +684,17 @@ public String getPairingText(String clientName, String clientContact, String cli
      */
     public String getPairingFileText(HashMap<Client, Property> clientPropertyPair) {
         String pairText = "";
-        for (Client clientText : clientPropertyPair.keySet()) {
+        for (Client client : clientPropertyPair.keySet()) {
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Storage.java:687]: Good catch

**Review by ngdeqi — COMMENTED** [review]
Some sequence diagram inaccuracies. PPP looks good.

File: docs/team/wilsonngjingan.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# Wilson Ng Jing An - Project Portfolio Page
```
Comment (OVReader) [inline docs/team/wilsonngjingan.md:1]: Nice format which enables easy reading

File: docs/team/wilsonngjingan.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+- **Enhancement to existing features:**
+   - Restructured Parser Class to multiple parser class that is specific to the commands. (Pull Request: [#87](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/87))
+   - Changing the PairingList to take in Object as a key instead of a String (Pull Request: [#69](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/69))
+   - Improve the consistency such that when a Client has been removed, the pairing will not be loaded. (Pull Request: [#58](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/58))
+   - Increase the branch coverage for Find Client and Property to 61% (Pull Request: [#193](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/193))
+
+- **Documentation:**
+
+  - User Guide:
+      - Added documentation for the features `Storage` and `find` [#135](https://github.com/AY2223S1-CS2113-F11-1/tp/pull/135)
```
Comment (OVReader) [inline docs/team/wilsonngjingan.md:41]: I like the use of hyperlinks to relevant issues to support your contribution claims. Very professional!

**Review by OVReader — APPROVED** [review]
Overall LGTM, just take note of the issues raised by @ngdeqi. Also, do consider adding a note in DG about the immediate object deletion (red cross) which occurs at the end of activation bar in sequence diagram.

**Pull request merged**